pages/common/waifu2x-ncnn-vulkan.md

```diff
@@ -0,0 +1,17 @@
+# waifu2x-ncnn-vulkan
+
+> Image upscaler for manga/anime-style images using NCNN neural network framework.
+> Runs fast on Intel/AMD/Nvidia/Apple-Silicon with Vulkan API.
+> More information: <https://github.com/nihui/waifu2x-ncnn-vulkan>.
+
+- Upscale image:
+
+`waifu2x-ncnn-vulkan -i {{path/to/input_file}} -o {{path/to/output_file}}`
+
+- Upscale image by a custom scale factor and denoise it:
```

Comment:
```suggestion
- Upscale an image by a custom scale factor and denoise it:
```

Reply:
what do we mean by denoise + it isn't a word

Reply:
From `--help`:
> -n noise-level       denoise level (-1/0/1/2/3, default=0)

This basically controls level of smoothing that happens with the image during upscaling. Higher value is more smoothing = less "noise" which can be e.g. artifacts from lossy compression.